src/main/kotlin/com/terraformation/backend/tracking/ObservationService.kt

```diff
@@ -408,38 +412,56 @@ class ObservationService(
   }
 
   /**
-   * Schedule an ad-hoc observation. This creates an ad-hoc observation, creates an ad-hoc
-   * monitoring plot, and adds the plot to the observation.
+   * Record an ad-hoc observation. This creates an ad-hoc observation, creates an ad-hoc monitoring
+   * plot, adds the plot to the observation, and complete the observation with plants.
    */
-  fun scheduleAdHocObservation(
-      endDate: LocalDate,
+  fun completeAdHocObservation(
+      conditions: Set<ObservableCondition>,
+      notes: String?,
+      observedTime: Instant,
       observationType: ObservationType,
       plantingSiteId: PlantingSiteId,
+      plants: Collection<RecordedPlantsRow>,
       plotName: String,
```

Comment:
Just noting that plot names are going away shortly, so we shouldn't build new client code that expects to be able to pass in a user-supplied name. (But removing this parameter isn't appropriate yet since names still exist for now.)